_layouts/sidenav.html

```diff
@@ -16,9 +16,16 @@
     <ul class="usa-sidenav">
       <li class="usa-sidenav__item">
         <!-- TODO: Parent item should not be clickable -->
-        <a href="{{ page.url | prepend: site.baseurl }}" class="usa-current"
-          >{{ site.data[page.lang].settings["nav"][page.sidenav]["title"] }}</a
-        >
+        <!-- If the user is on the policy page, make the parent item fixed -->
+        {% if page.sidenav == "policies" %}
+          <a href="{{ '/policy/' | locale_url }}" class="usa-current"
+            >{{ site.data[page.lang].settings["nav"][page.sidenav]["title"] }}</a
+          >
+        {% else %}
+          <a href="{{ page.url | prepend: site.baseurl }}" class="usa-current"
+            >{{ site.data[page.lang].settings["nav"][page.sidenav]["title"] }}</a
+          >
+        {% endif%}
```

Comment:
Instead of hard-coding a specific behavior for the policies page, what if we made it so that each sidenav should (or at least could) define a url for it's top-level section?

Something like `url: /policy/` in the section here, after `title`:

https://github.com/18F/identity-site/blob/33c3bc8307a0cc993212009e5977d8a59145c58b/_data/en/settings.yml#L209-L210

Then we could use that as the top-level link:

```suggestion
        {% assign section_url = site.data[page.lang].settings.nav[page.sidenav].url | default: page.url %}
        <a href="{{ section_url | prepend: site.baseurl }}" class="usa-current">
          {{ site.data[page.lang].settings.nav[page.sidenav].title }}
        </a>
```